cockatrice/src/localserver.cpp

```diff
@@ -6,7 +6,7 @@ LocalServer::LocalServer(QObject *parent)
     : Server(parent)
 {
     setDatabaseInterface(new LocalServer_DatabaseInterface(this));
-    addRoom(new Server_Room(0, 0, QString(), QString(), QString(), false, QString(), QStringList(), this));
+    addRoom(new Server_Room(0, 0, QString(), QString(), QString(), QString(), false, QString(), QStringList(), this));
```

Comment:
No it does not.  Check out the localserver.h  I can't say I have dug into why this is for the local server.